### PR TITLE
Add a lot more post info to readme

### PR DESCRIPTION
**Pull request by ct-martin**
Re: #253 
Related: RITlug/runbook#78
cc @jrtechs @Tjzabel @jwflory 

This makes the README fairly long, but CONTRIBUTING was at some point designated to be for more dev-oriented things about the site IIRC...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,13 +36,237 @@ Asset file names should have enough info in the file name to know what reference
...
+date-end: "2020-04-25 18:00"
+location: "TBD"
+title: "ImagineRIT"
+---
+Help or visit RITlug at ImagineRIT!
+
+Stay tuned for more info closer to the event.
+```
+
+Placeholders can also take an `rrule` frontmatter for recurring events, written to the iCal spec.
```
Comment (jrtechs) [inline README.md]: Could we link to how we can format the rrule.

File: README.md [hunk 2]
```diff
@@ -27,13 +36,237 @@ Asset file names should have enough info in the file name to know what reference
...
+date-start: "2019-12-06 16:30"
+date-end: "2019-12-06 18:00"
+location: "GOL-2620 (Med. DB Lab)"
+---
+```
+
+General Announcement:
+```
+---
+title: title of your announcement
```
Comment (jrtechs) [inline README.md]: Nice examples for each post type

**Comment by Tjzabel** [comment]
@ct-martin @jrtechs so I'm okay with these additions, but I'm wondering if this information would be better suited for a CONTRIBUTING.md doc?

**Comment by jrtechs** [comment]
@Tjzabel the contributing document appears to currently be more of a "development" document where the readme has information on how to push new content to the website.

**Comment by ct-martin** [comment]
@tjzabel as stated, CONTRIBUTING was at one point intended to be dev-related, and things like talks are often done by people outside RITlug, I think this is better to keep in README. If we want to re-evaluate that later I'm open to it being discussed in a separate issue

**Comment by ct-martin** [comment]
(since I got asked elsewhere, `CONTRIBUTING` is a markdown file similar to `README`; it's not changed in this issue, but its purpose is related/relevant)

**Comment by Tjzabel** [comment]
@ct-martin the branch says it's out of date though. Can you fix that before this is merged?

**Comment by ct-martin** [comment]
Done :+1:

**Pull request merged**